chalicelib/checks/audit_checks.py

```diff
@@ -349,6 +349,7 @@ def check_help_page_urls(connection, **kwargs):
                 # test only link part of match (not text part, even if it looks like a link)
                 idx = link.index(']')
                 url = link[link.index('(', idx)+1:-1]
+                urls.append(url)
```

Comment:
Thanks for adding this, could you also change line 338 (`results = ...`) to use `&q=help&q=resources` instead of just `&q=help` ?